src/components/TodoDetails.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div>
+    <page-header></page-header>
+    <p> {{ getItems[index].name }} </p>
+    <home-button></home-button>
+  </div>
+</template>
+
+<script>
+  import { Vue, Component, Prop} from 'vue-property-decorator';
+  import {mapGetters} from 'vuex';
+  import PageHeader from './PageHeader.vue';
+  import HomeButton from './HomeButton';
+
+  @Component({
+    components: {
+      pageHeader: PageHeader,
+      homeButton: HomeButton
+    },
+    computed: {
+        ...mapGetters([
+            'getItems'
```

Comment:
Instead of this, you could import only the getItems getter to this component, and create a method like this: get items(index) {return getItems[index].name}.